homeassistant/components/comelit/config_flow.py

```diff
@@ -23,8 +25,10 @@ def user_form_schema(user_input: dict[str, Any] | None) -> vol.Schema:
     user_input = user_input or {}
     return vol.Schema(
         {
-            vol.Optional(CONF_HOST, default=DEFAULT_HOST): str,
-            vol.Optional(CONF_PIN, default=DEFAULT_PIN): str,
+            vol.Required(CONF_HOST, default=DEFAULT_HOST): cv.string,
+            vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
+            vol.Optional(CONF_PIN, default=DEFAULT_PIN): cv.positive_int,
+            vol.Required(CONF_DEVICE, default=BRIDGE): vol.All(BRIDGE),
```

Comment:
I think we should remove this for now, and add it when it becomes relevant. 

A choice between 1 possible item, is no choice (and thus unrelated to this PR as well).

Reply:
Done